Tests/DataPipeline/DataPipelineInteractionTests.swift

```diff
@@ -112,8 +104,6 @@ class CustomerIOImplementationTest: UnitTest {
 
         customerIO.identify(identifier: givenIdentifier)
 
-        XCTAssertFalse(hooksMock.mockCalled)
-
```

Comment:
Wondering that we no longer make use of hooks in this test method but the name still says `test_identify_givenProfileAlreadyIdentified_expectDoNotRunHooks_expectDoNotDeleteDeviceToken`

Reply:
See my [reply above](https://github.com/customerio/customerio-ios/pull/473#discussion_r1448462344)